docs/source/caching.rst

```diff
@@ -0,0 +1,258 @@
+=======
+Caching
+=======
+
+Resolve Caching
+===============
+
+Resolve caching is a feature that caches resolves to a `memcached <https://memcached.org/>`_ (in-memory) data store.
+Because the data store is in-memory, the full contents of the cache are lost if the memcached service shuts down
```

Comment:
I think I would move this in the pros/cons paragraph.

Reply:
@JeanChristopheMorinPerso Sorry what do you mean?

Reply:
Sorry, I wasn't very clear. I mean maybe move this to paragraph at line 18?